packages/grafana-data/src/valueFormats/valueFormats.ts

```diff
@@ -230,6 +240,16 @@ export function getValueFormat(id?: string | null): ValueFormatter {
       if (key === 'currency') {
         return currency(sub);
       }
+
+      if (key === 'bool') {
+        idx = sub.indexOf('/');
+        if (idx >= 0) {
+          const t = sub.substring(0, idx);
+          const f = sub.substring(idx + 1);
+          return booleanValueFormatter(t, f);
+        }
+        return booleanValueFormatter(sub, '-');
+      }
```

Comment:
I read this as being able to do: `bool:ON/OFF`  ?  can we add this to https://github.com/grafana/grafana/blob/main/docs/sources/panels/field-options/standard-field-options.md#custom-units